src/main/kotlin/lotto/model/game/Lottos.kt

```diff
@@ -1,49 +1,49 @@
-package lotto.model
+package lotto.model.game
 
+import lotto.model.result.Coincidence
 import java.math.BigDecimal
 import java.math.RoundingMode
 
-class Lottos(private val lottos: List<Lotto>) {
-    var myLottos: List<Lotto>
-        private set
-
-    init {
-        myLottos = lottos
-    }
-
+class Lottos(val lottos: List<Lotto>) {
     constructor(lottoNumPool: LottoNumberPool, count: Int) : this(createLottos(lottoNumPool, count))
 
     fun check(winningLotto: Lotto, checkCount: Int): Int {
```

Comment:
`Lottos` 외에 사용하는 곳이 없어 보이네요. public이 아닌 private으로 바꾸면 어떨까요?

```suggestion
    private fun check(winningLotto: Lotto, checkCount: Int): Int {
```